Common.props

```diff
@@ -5,7 +5,7 @@
     <Authors>Sebastien Ros</Authors>
     <PackageProjectUrl>https://github.com/sebastienros/fluid</PackageProjectUrl>
     <PackageLicenseUrl>https://github.com/sebastienros/fluid/blob/master/LICENSE.txt</PackageLicenseUrl>
-    <TargetFrameworks>netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
What's the reason?

Reply:
It's because of the `.IsGenericType` usage [here](https://github.com/sebastienros/fluid/pull/182/files#diff-166c73ae62ad714b7187c2df958a14c6R44). NET 4.6 has it but netstandard1.6 doesn't.

I could make an argument for updating to netstandard2.0 but I thought I'd do this because it keeps the code simple going forward.

I can change it to be 1.6 compatible if needed.